kubernetes/zenko/values.yaml

```diff
@@ -76,6 +76,11 @@ cloudserver:
     # accountName:
       # access:
       # secret:
+  # externalBackends:
+  #   aws_s3:
+  #     keepAlive: false
+  #   gcp:
+  #     keepAlive: true
```

Comment:
IMO, there is no need to put values in the Zenko chart when the defaults from the subchart don't neet to be overwritten.